bash.html.markdown

```diff
@@ -25,7 +25,21 @@ Nearly all examples below can be a part of a shell script or executed directly i
 # As you already figured, comments start with #. Shebang is also a comment.
 
 # Simple hello world example:
-echo Hello world!
+echo "Hello world!"
+
+# Color output of echo command
+# This command will show green 'Hello world!' text
+echo -e "\033[0;32mHello world!\033[0m"
+
+# You can use these codes
+Black        0;30     Dark Gray     1;30
```

Comment:
Right but `Black` isn't a command on most machines, nor are all the other color names. It's nice to have the color codes listed here, but they should be as comments.